syft/pkg/dart_pub_metadata.go

```diff
@@ -0,0 +1,42 @@
+package pkg
+
+import (
+	"github.com/anchore/packageurl-go"
+	"github.com/anchore/syft/syft/linux"
+)
+
+type DartPubMetadata struct {
+	Name        string   `mapstructure:"name" json:"name"`
+	Version     string   `mapstructure:"version" json:"version"`
+	Description string   `mapstructure:"description" json:"description,omitempty"`
+	Authors     []string `mapstructure:"authors" json:"authors,omitempty"`
```

Comment:
nit: we try not to `omitempty` collections, and instead enforce an empty collection

Reply:
I think this is actually extraneous and can be removed altogether.